### ISSUE TITLE
How to disable antialiasing (everything appears bold)?

### ISSUE THREAD

**Issue by jaraco** [zeit/hyper#2714]
- [x] I am on the [latest](https://github.com/zeit/hyper/releases/latest) Hyper.app version
- [x] I have searched the [issues](https://github.com/zeit/hyper/issues) of this repo and believe that this is not a duplicate

- **macOS 10.13.3**:
- **Hyper.app version**: 1.4.8
- **[.hyper.js](https://gist.github.com/f3096b54d0ac14e58639c35aaf95afca)**
- **Relevant information from devtools** _(CMD+ALT+I on macOS, CTRL+SHIFT+I elsewhere)_: Inspecting the x-row element shows `font-weight: lighter`.
- **The issue is reproducible in vanilla Hyper.app**: Is Vanilla

## Issue

None of the options in #175 seem to have any effect. All text renders as bold. I've confirmed I can change the color of the font, but I seem to be unable to affect the weight of the font. Here's a screenshot of Hyper over Terminal using the same font:

![image](https://user-images.githubusercontent.com/308610/36683146-d4445ff2-1aea-11e8-8fc8-45d9cf5963b2.png)

Obviously, there are some features pending a new release, but I should think it's possible in 1.4.8 to not have everything appear in bold. What more can I do to troubleshoot?

**Comment by chabou** [comment]
In our last canary, there are 2 new options to disable bold font.
https://github.com/zeit/hyper/blob/8ebdaebdfa2199f6f957d4aa94425f60fa65bc9e/app/config/config-default.js#L17-L21

If font is still enough thin, I will suggest you to change used font.

**Comment by jaraco** [comment]
Thanks @chabou.

I tried `fontWeight: 'normal'`, which has no effect. `fontWeight: 'bold'` makes the font slightly more bold.

> If font is still enough thin, I will suggest you to change used font.

:/

But I can see this font, Monaco, in the Terminal app, able to render with a more 'normal' weight.

I have the same issue with the default fontFamily:

```
    fontFamily: 'Menlo, "DejaVu Sans Mono", Consolas, "Lucida Console", monospace',
```

Indeed, if I remove Menlo from that list, I get a font that renders not so bold.

I noticed I get the same bold look also in Terminal if I enable 'Antialias text'. So maybe what I'm seeking in this ticket is to disable antialiasing.

I tried this:

```
    termCSS: 'font-smooth: never; -webkit-font-smoothing: none;',
```

But that seemed to have no effect.

I tried using another font I have installed, something with a "Thin" version, but Hyper doesn't seem to find it. I'll file a separate ticket about that.

Any suggestions on disabling antialiasing or otherwise getting a thinner rendering of Menlo or Monaco?
